### PR TITLE
Create service for sending reports

### PR DESCRIPTION
**Pull request by rkondratowicz**
This PR just adds a service (and tests) that will be run periodically in the future.
Also smtp credentials and recipients will be configured in future PR.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSender.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+public class ReportSender {
+
+    private static final Logger log = LoggerFactory.getLogger(ReportSender.class);
+
+    public static final String EMAIL_SUBJECT = "Bulk Scan daily report";
+    public static final String EMAIL_BODY = "This is an auto generated email. Do not respond to it.";
+    public static final String ATTACHMENT_PREFIX = "bulk_scan_envelopes_";
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSender.java]: suggestions for alternative email title/body/attachment welcome

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSender.java]: `zip-file-summary-`?

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSenderTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import javax.mail.Address;
+
+import static java.time.LocalDate.now;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReportSenderTest {
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSenderTest.java]: Would be nice to add tests to cover `MessagingException` and `IOException`

Comment (rkondratowicz) [reply src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSenderTest.java]: added extra test

**Pull request merged**